lms/templates/email/instructor_email_digest/subject.jinja2

```diff
@@ -0,0 +1 @@
+There has been some activity in Hypothesis
```

Comment:
The subject is just one line so it could just be a Python f-string rather than a separate file. But when it comes to things like internationalisation, localisation, and pluralisation, it's nice to have the same Jinja2 stuff available for the subject as for the body template. Of course this subject doesn't do any internationalisation, localisation, or pluralisation but I think it might in future.